people_msgs/msg/ObjectInMask.msg

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This message can represent a detected object and its region of interest
+string object_name  				# object name
+float32 probability 				# probability of detected object
+sensor_msgs/RegionOfInterest roi    # region of interest
+sensor_msgs/Image mask				# Instance mask as Image
```

Comment:
what's the size of the image? Is it the same as roi.size()?
For the data element inside mask, is it stand for possibility of category(object_name)?

Reply:
Hi Challen, the file above is outdated. This is the updated [ObjectInMask.msg](https://github.com/intel/ros2_openvino_toolkit/blob/4156cb187cc3b4cc137e13fe1a5d848b90f41888/people_msgs/msg/ObjectInMask.msg) file, where **sensor_msgs/Image** is replaced by **float32[]** .

The mask (float32[]) has the same size with roi.size() and it stores every pixel's possibility of being the object.

Reply:
> Hi Challen, the file you provide above is outdated. This is the updated [ObjectInMask.msg](https://github.com/intel/ros2_openvino_toolkit/blob/4156cb187cc3b4cc137e13fe1a5d848b90f41888/people_msgs/msg/ObjectInMask.msg) file, where **sensor_msgs/Image** is replaced by **float32[]** .
> 
> The mask (float32[]) has the same size with roi.size() and it stores every pixel's possibility of being the object.

Then it should be ok for me.